modules/registry-operator-distribution-across-availability-zones.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-distribution-across-availability-zones_{context}"]
+= Image Registry Operator distribution across availability zones
+
+The default configuration of the Image Registry Operator spreads image registry pods across topology zones to prevent delayed recovery times in case of a complete zone failure where all pods are impacted. 
+
+The Image Registry Operator defaults to the following when deployed with a zone related topology constraint:
+
+.Image Registry Operator deployed with a zone related topology constraint
+[source,yaml]
+----
+  topologySpreadConstraints:
+  - labelSelector:
+      matchLabels:
+        docker-registry: default
+    maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: DoNotSchedule
+  - labelSelector:
+      matchLabels:
+        docker-registry: default
+    maxSkew: 1
+    topologyKey: node-role.kubernetes.io/worker
+    whenUnsatisfiable: DoNotSchedule
+  - labelSelector:
+      matchLabels:
+        docker-registry: default
+    maxSkew: 1
+    topologyKey: topology.kubernetes.io/zone
+    whenUnsatisfiable: DoNotSchedule
+----
+
+The Image Registry Operator defaults to the following when deployed without a zone related topology constraint, which applies to bare metal and vSphere instances:
+
+.Image Registry Operator deployed without a zone related topology constraint
+[source,yaml]
+----
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        docker-registry: default
+  - maxSkew: 1
+    labelSelector:
+      matchLabels:
+        docker-registry: default
+    topologyKey: node-role.kubernetes.io/worker
+    whenUnsatisfiable: DoNotSchedule
```

Comment:
@bmcelvee Sorry for my mistake, it should be 
```
 topologySpreadConstraints:
  - labelSelector:
      matchLabels:
        docker-registry: default
    maxSkew: 1
    topologyKey: kubernetes.io/hostname
    whenUnsatisfiable: DoNotSchedule
  - labelSelector:
      matchLabels:
        docker-registry: default
    maxSkew: 1
    topologyKey: node-role.kubernetes.io/worker
    whenUnsatisfiable: DoNotSchedule
```

Reply:
@bmcelvee Above comment need address, thanks

Reply:
Yes, thanks! I'm updating it now.

Reply:
Updated!